### PR TITLE
Article: Still WIP :-(

### PR DESCRIPTION
**Pull request by paluh**

### CODE REVIEW

File: docs/design/bl.md [hunk 1]
```diff
@@ -1,85 +1,625 @@
...
+note that once Lightning Channel is established payments exectued on that layer
+achieve nearly instant payment finality. Successful payment paths ensure that
+transactions are finalized predominantly by network speed.
+
+#### 3.2 Data Availability and Lightweight Clients
+
+##### 3.2.1 BLN "lightweight" clients
+
+Blockchain data availability is crucial for the safety of the BLN channels. The
+main channel UTxO's on-chain presence ensures that Layer 2 operations are secure
```
Comment (waalge) [inline docs/design/bl.md:175]: Is this not L2? 

If there is a system already in place, like long hand for first occurence in each section, then just ignore me

File: docs/design/bl.md [hunk 2]
```diff
@@ -1,85 +1,625 @@
...
+dynamic nature of user participation and improve operational efficiency.
+
+### 4.3 Penalty System - Design Choice or Imposed Limitation
+
+#### 4.3.1 Overview of BLN's Penalty System
+
+The penalty system in the Bitcoin Lightning Network (BLN) acts as a safeguard
+against dishonest behaviors, specifically attempts to close the channel with an
+outdated state. To deter such actions, both parties in a transaction associate
+every historical transaction with a "revocation key." This key can penalize the
```
Comment (waalge) [inline docs/design/bl.md:504]:
```suggestion
every historical transaction with a "revocation key". This key can penalize the
```

File: docs/design/bl.md [hunk 3]
```diff
@@ -1,85 +1,625 @@
...
+counterparty if they attempt to publish an outdated transaction. Additionally,
+the protocol requires that a reserve amount be maintained at all times,
+providing a financial disincentive against dishonesty. This system is often
+justified through game-theoretical arguments, suggesting that the fear of losing
+the reserve should theoretically deter fraud.
+
+#### 4.3.2 Critique and Alternatives to the Penalty System
+
+While effective in deterring fraud, the penalty system introduces significant
+complexity and operational overhead. It necessitates the locking up of funds and
```
Comment (waalge) [inline docs/design/bl.md:514]: Citation? 
https://delvingbitcoin.org/t/ln-symmetry-project-recap/359

File: docs/design/bl.md [hunk 4]
```diff
@@ -1,85 +1,625 @@
...
+[`SIGHASH_ANYPREVOUT`](https://github.com/bitcoin/bips/blob/master/bip-0118.mediawiki)),
+simplifies the protocol by removing the need for penalties and reducing storage
+requirements. However, it relies on capabilities not currently available in
+Bitcoin script, such as a new opcode.
+
+##### Benefits of Eltoo:
+
+- **Reduced Storage Requirements**: Eliminates the need to store revocation
+  secrets.
+- **Symmetry of the Protocol**: Removes "toxic information," simplifying the
```
Comment (waalge) [inline docs/design/bl.md:535]:
```suggestion
- **Symmetry of the Protocol**: Removes "toxic information", simplifying the
```

File: docs/design/bl.md [hunk 5]
```diff
@@ -1,85 +1,625 @@
...
+#### 4.3.3 Cardano's Approach: Beyond Penalties
+
+On Cardano Lightning, the unique capabilities of the platform could support an
+alternative approach that avoids the traditional penalty system. Inspired by
+mechanisms like Eltoo and Hydra, CL could allow for the direct overriding of
+outdated states without imposing penalties. This method would simplify channel
+management by eliminating the need for revocation keys and reducing operational
+complexity.
+
+##### Optional Penalty System:
```
Comment (waalge) [inline docs/design/bl.md:549]:
```suggestion
##### Optional Penalty System
```

**Review by waalge — APPROVED** [review]

File: docs/design/bl.md (hunk 1 above)
Comment (paluh) [reply docs/design/bl.md:175]: You are right - I went too far probably with this whole section as it mainly relates to the L2. Can I leave it as we have in the intro that we will somewhat explore that area as well?

**Comment by paluh** [comment]
Closing this as the article was added to the blog.

**Pull request closed without merging**